modules/programs/firefox.nix

```diff
@@ -731,8 +734,11 @@ in {
     home.file = mkMerge ([{
       "${firefoxConfigPath}/profiles.ini" =
         mkIf (cfg.profiles != { }) { text = profilesIni; };
-      "${mozillaConfigPath}/native-messaging-hosts".source =
-        "${nativeMessagingHostsJoined}/lib/mozilla/native-messaging-hosts";
+      "${mozillaConfigPath}/native-messaging-hosts" = {
+        source =
+          "${nativeMessagingHostsJoined}/lib/mozilla/native-messaging-hosts";
```

Comment:
The issue with browserpass is that it symlinks to a location in `${nativeMessagingHostsJoined}/lib/browserpass`, since we're now doing `recursive` below, it means that the path will not be available and the symlink is invalid. Not really sure what to do here.

Reply:
Yep I've noticed that too, I'm looking for a solution :)

Reply:
https://github.com/NixOS/nixpkgs/pull/284309

Call me radical 😄 @andresilva